src/Header/Header.scss

```diff
@@ -16,11 +27,16 @@
     }
   }
 
-
   a {
     outline: none;
     text-decoration: none;
     text-align: center;
-    font-size: 1.8rem
+    font-size: 1.8rem;
+  }
+
+  .view-saved {
```

Comment:
Thanks for adding these hovers. The available user actions are much more clear now.